config/docker_compose.toml

```
@@ -155,6 +155,7 @@ iatapay.base_url = "https://sandbox.iata-pay.iata.org/api/v1"
 inespay.base_url = "https://apiflow.inespay.com/san/v21"
 itaubank.base_url = "https://sandbox.devportal.itau.com.br/"
 jpmorgan.base_url = "https://api-mock.payments.jpmorgan.com/api/v2"
+juspaythreedsserver.base_url = "http://localhost:8000"
 jpmorgan.secondary_base_url="https://id.payments.jpmorgan.com"
```

```suggestion
jpmorgan.secondary_base_url = "https://id.payments.jpmorgan.com"
```
nit: consistant formatting